src/collections/_documentation/workflow/integrations/legacy-integrations.md

```diff
@@ -117,21 +117,24 @@ heroku labs:enable runtime-dyno-metadata -a <app name>
 
 This exposes the `HEROKU_SLUG_COMMIT` environment variable, which most Sentry SDKs will automatically detect and use for configuration.
 
-Next you’ll want to add your repository and setup a deploy hook.
+Next, you’ll want to add your repository and setup a deploy hook:
 
-1.  Start by connecting your repository to your Sentry organization so that we can automatically retrieve your commit data.
+1.  Start by connecting your repository to your Sentry organization so that we can automatically retrieve your commit data. To do this find your repository integration (e.g. GitHub, GitLab, Bitbucket, etc.) in Settings > Integrations and from the "Configurations" tab click "Configure".
```

Comment:
Generally good to avoid needless words to make it easier for readers and avoid Latin terms (e.g, i.e., etc.). Also bold menu paths.

Suggest "Start by connecting your repository to your Sentry organization so we automatically retrieve your commit data. Find your repository integration (GitHub, GitLab, Bitbucket, for example) in **Settings > Integrations **. From the **Configurations** tab, click "Configure"."